CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ include(CheckCXXSourceCompiles)
 include(CheckCXXSourceRuns)
 
 set(CMAKE_C_FLAGS "-std=gnu99")
-set(CMAKE_CXX_FLAGS "-ansi -pedantic -Wall -Wextra -Wno-long-long")
+set(CMAKE_CXX_FLAGS "-ansi -pedantic -Wall -Wextra -Wno-long-long -std=gnu++11")
```

Comment:
This was presumably made to resolve a build issue, correct?

Reply:
Correct (I don't remember what the issue was, though).